email/registration.html

```diff
@@ -0,0 +1,26 @@
+<div>
+  <p>All Glories to Sri Sri Guru and Gauranga!</p>
+  <p>Hare Krsna. Dandavat pranams dear {{2.your_name}}.</p>
```

Comment:
`2.your_name` will not work. You should refer to the data passed into the template through `params`. See [below](https://github.com/PureBhaktiArchive/audioseva/pull/71/files#diff-3af462e09d881812253c21aa19cc1fa4R21).